learning/step_by_step/scripting.rst

```diff
@@ -28,7 +28,7 @@ to the high integration with Godot:
 * It's simple, elegant and designed to be friendly with users of other languages such as Lua, Python, Squirrel, etc.
 * Loads and compiles blazingly fast.
 * The editor integration is a pleasure to work with, with code completion expanding as far as completing nodes, signals, and many other items pertaining to the current scene being edited.
-* Has vector types built-in (such as Vectors, transorms etc), making it efficient for heavy linear algebra.
+* Has vector types built-in (such as Vectors, transforms etc), making it efficient for heavy linear algebra.
```

Comment:
While at it, please add a comma before `etc`, and a dot afterwards (so `transforms, etc.)`).

Reply:
Ow, too late! Didn't see that one. Sorry!